tests/integration/test_array.py

```diff
@@ -11,6 +11,7 @@
 import guppylang.prelude.quantum as quantum
 
 
+@pytest.mark.xfail(reason="hugr-includes-whole-stdlib")
```

Comment:
Maybe we should just comment out the asserts below instead of marking the whole test as xfail? Then we'd at least still be testing the compilation code paths and validation

Reply:
Done with explicit call to `pytest.xfail` so we have a good marker, will document in issue